src/Components/SignUpForm/SignUpForm.js

```diff
@@ -1,14 +1,23 @@
-import React from 'react'
+import { Component} from 'react'
 import SignIn from '../../Containers/SignIn/SignIn'
 import SignUp from '../../Containers/SignUp/SignUp'
 
-const SignUpForm = () => {
+class SignUpForm extends Component {
+  constru() {
```

Comment:
is this a shorthand for constructor?

Reply:
Nope that was definitely just an oversight. I just fixed it and will push it in my next merge